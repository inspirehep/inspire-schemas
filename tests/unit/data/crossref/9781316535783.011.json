{
   "status":"ok",
   "message-version":"1.0.0",
   "message":{
      "DOI":"10.1017/9781316535783.011",
      "subtitle":[

      ],
      "issued":{
         "date-parts":[
            []
         ]
      },
      "publisher-location":"Cambridge",
      "short-title":[

      ],
      "prefix":"10.1017",
      "relation":{

      },
      "ISBN":[
         "9781316535783"
      ],
      "author":[
         {
            "affiliation":[

            ],
            "given":"Chris",
            "family":"Smeenk"
         }
      ],
      "reference-count":0,
      "member":"56",
      "source":"Crossref",
      "score":1.0,
      "deposited":{
         "timestamp":1523597812000,
         "date-time":"2018-04-13T05:36:52Z",
         "date-parts":[
            [
               2018,
               4,
               13
            ]
         ]
      },
      "editor":[
         {
            "affiliation":[

            ],
            "given":"Khalil",
            "family":"Chamcham"
         },
         {
            "affiliation":[

            ],
            "given":"Joseph",
            "family":"Silk"
         },
         {
            "affiliation":[

            ],
            "given":"John D.",
            "family":"Barrow"
         },
         {
            "affiliation":[

            ],
            "given":"Simon",
            "family":"Saunders"
         }
      ],
      "indexed":{
         "timestamp":1524067539055,
         "date-time":"2018-04-18T16:05:39Z",
         "date-parts":[
            [
               2018,
               4,
               18
            ]
         ]
      },
      "type":"book-chapter",
      "URL":"http://dx.doi.org/10.1017/9781316535783.011",
      "is-referenced-by-count":0,
      "references-count":0,
      "original-title":[

      ],
      "short-container-title":[

      ],
      "publisher":"Cambridge University Press",
      "content-domain":{
         "domain":[

         ],
         "crossmark-restriction":"False"
      },
      "created":{
         "timestamp":1492423296000,
         "date-time":"2017-04-17T10:01:36Z",
         "date-parts":[
            [
               2017,
               4,
               17
            ]
         ]
      },
      "title":[
         "Testing Inflation"
      ],
      "container-title":[
         "The Philosophy Of Cosmology"
      ],
      "page":"206-227"
   },
   "message-type":"work"
}
