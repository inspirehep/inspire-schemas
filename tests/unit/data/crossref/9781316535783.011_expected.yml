---
dois:
- doi: 10.1017/9781316535783.011
  material: publication
title: Testing Inflation
authors:
- full_name: Smeenk, Chris
page_start: '206'
page_end: '227'
year:
material: publication
parent_isbn: '9781316535783'
document_type: book chapter
imprints:
